xds/src/test/java/io/grpc/xds/XdsClientImplDataTest.java

```diff
@@ -172,7 +172,6 @@ public class XdsClientImplDataTest {
   @Before
   public void setUp() {
     originalEnableRouteLookup = XdsResourceType.enableRouteLookup;
-    assertThat(originalEnableRouteLookup).isFalse();
```

Comment:
Would you want to assert that it's `true`?

Reply:
It seemed to me to just be clutter to check the default.  It made some sense when it was a protective flag, but checking for true on every flag we've ever introduced seems messy to me.